android/src/main/java/com/pspdfkit/flutter/pspdfkit/ConfigurationAdapter.java

```diff
@@ -0,0 +1,370 @@
+        }
+    }
+
+    private void configureShowShareAction(boolean showShareAction) {
+        if (showShareAction) {
+            configuration.setEnabledShareFeatures(ShareFeatures.all());
+        } else {
+            configuration.setEnabledShareFeatures(ShareFeatures.none());
+        }
+    }
+
+    private void configureShowPrintAction(boolean showPrintAction) {
+        if (showPrintAction) {
+            configuration.enablePrinting();
+        } else {
+            configuration.disablePrinting();
+        }
+    }
+
+    private void configureEnableTextSelection(boolean enableTextSelection) {
+        configuration.textSelectionEnabled(enableTextSelection);
+    }
+
+    private void configureDocumentInfoView(boolean enableDocumentInfoView) {
+        if (enableDocumentInfoView) {
+            configuration.enableDocumentInfoView();
+        } else {
+            configuration.disableDocumentInfoView();
+        }
+    }
+
+    private void configureThemeMode(String themeMode) {
+        ThemeMode result = ThemeMode.DEFAULT;
+        if (themeMode.equals(APPEARANCE_MODE_DEFAULT)) {
+            result = ThemeMode.DEFAULT;
+        } else if (themeMode.equals(APPEARANCE_MODE_NIGHT)) {
+            result = ThemeMode.NIGHT;
+        }
+        configuration.themeMode(result);
+    }
+
+    private void configureDarkThemeRes(String darkThemeResource, Context context) {
+        @StyleRes int darkThemeId = getStyleResourceId(darkThemeResource, context);
+        if (darkThemeId != 0) {
+            configuration.themeDark(darkThemeId);
+        }
+    }
+
+    private void configureDefaultThemeRes(String defaultThemeResource, Context context) {
+        @StyleRes int defaultThemeId = getStyleResourceId(defaultThemeResource, context);
+        if (defaultThemeId != 0) {
+            configuration.theme(defaultThemeId);
+        }
+    }
+
+    private static int getStyleResourceId(String styleName, Context context) {
+        int resourceId = context.getResources().getIdentifier(styleName, "style", context.getPackageName());
+        if (resourceId == 0) {
+            Log.e(LOG_TAG, String.format("Style resource not found for %s", styleName));
+        }
+        return resourceId;
+    }
+
+    private <T> boolean containsKeyOfType(HashMap<String, Object> configurationMap, String key, Class<T> clazz) {
+        if (configurationMap.get(key) != null) {
+            checkCast(configurationMap.get(key), clazz, key);
+            return true;
+        }
+        return false;
+    }
+
+    private static <T> void checkCast(Object object, Class<T> clazz, String key) {
+        if (!clazz.isInstance(object)) {
+            throw new ClassCastException(String.format("Value for the key %s must be of type %s.", key, clazz.getName()));
+        }
+    }
+
+    PdfActivityConfiguration build() {
+        return configuration.build();
+    }
+
+    static PdfActivityConfiguration.Builder getDefaultConfiguration(Context context) {
```

Comment:
Why do we actually provide this as default configuration over the defaults inherent to `PdfActivityConfiguration`? Are these defaults shared between the Android and iOS implementations?

Reply:
Same reason here, to make it consistent with the react-native wrapper https://github.com/PSPDFKit/react-native/blob/a5e24ca21b360e63ab9468b356792eded8db4b93/android/src/main/java/com/pspdfkit/react/ConfigurationAdapter.java#L304-L324

Reply:
I'd say we have to fix it here _and_ in RN then.